_posts/2020-10-01-Fortran-Newsletter-October-2020.md

```diff
@@ -0,0 +1,189 @@
+* [#180](https://github.com/fortran-lang/fpm/issues/180): Recursive source discovery
+* [#178](https://github.com/fortran-lang/fpm/issues/178): Add more example packages
+* [#177](https://github.com/fortran-lang/fpm/issues/177): Allow selective testing of single suites and tests
+* [#175](https://github.com/fortran-lang/fpm/issues/175): Updated formatting of Markdown documents
+* [#174](https://github.com/fortran-lang/fpm/issues/174): Cache Haskell Stack build in CI
+* [#171](https://github.com/fortran-lang/fpm/issues/171): Increase test coverage of fpm manifest
+* [#170](https://github.com/fortran-lang/fpm/issues/170): Source parsing tests
+* [#163](https://github.com/fortran-lang/fpm/issues/163): Use different strategy to fetch git dependencies
+* [#162](https://github.com/fortran-lang/fpm/issues/162): Updated OS type identification
+* [#160](https://github.com/fortran-lang/fpm/issues/160): Add contributing guidelines
+(you can read them [here](https://github.com/fortran-lang/fpm/CONTRIBUTING.md))
+* [#157](https://github.com/fortran-lang/fpm/issues/157): Implement reading of fpm.toml
+* [#155](https://github.com/fortran-lang/fpm/issues/155): Internal dependencies and build backend
+
+
+Work in progress:
+
+* [#193](https://github.com/fortran-lang/fpm/issues/193) (WIP): Local path dependencies
+* [#190](https://github.com/fortran-lang/fpm/issues/190) (WIP): Auto discovery of executables
+* [#189](https://github.com/fortran-lang/fpm/issues/189) (WIP): Implement `fpm new`
+
+fpm is still in early development and we need as much help as we can get.
+Here's how you can help today:
+
+* Try to use it. Does it work? No? Let us know! Read the [fpm packaging guide](https://github.com/fortran-lang/fpm/blob/master/PACKAGING.md) to learn how to build your package with fpm.
+* Browse the [open issues](https://github.com/fortran-lang/fpm/issues) and see if you can help implement any fixes or features.
+* Adapt your Fortran package for fpm and submit it to the [Registry](https://github.com/fortran-lang/fpm-registry).
+* Improve the documentation.
+
+The short term goal of fpm is to make development and installation of Fortran packages with dependencies easier.
+Its long term goal is to build a rich and decentralized ecosystem of Fortran packages and create a healthy
+environment in which new open source Fortran projects are created and published with ease.
+
+##  Fortran benchmarks
+
+We created the [benchmarks repository](https://github.com/fortran-lang/benchmarks) with the goal to design and implement a comprehensive set of benchmarks.
+The benchmarks will aim to compare the performance of various Fortran compilers, as well as the performance of canonical algorithms implemented in Fortran and different languages.
+If you'd like to contribute in any way, be it the design, implementation, or testing of benchmarks, please join the ongoing discussion [here](https://github.com/fortran-lang/benchmarks/issues).
```

Comment:
This was already reported in the previous newsletter (https://fortran-lang.org/newsletter/2020/08/01/Fortran-Newsletter-August-2020/), so it should be removed from here, unless there are more news.